controllers/idler/idler_controller_test.go

```diff
@@ -583,7 +583,195 @@ func TestEnsureIdlingFailed(t *testing.T) {
 		require.EqualError(t, err, "failed to ensure idling 'john-dev': cannot set status to fail")
 	})
 }
+func TestAppNameType(t *testing.T) {
 
+	idler := &toolchainv1alpha1.Idler{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "alex-stage",
+			Labels: map[string]string{
+				toolchainv1alpha1.SpaceLabelKey: "alex",
+			},
+		},
+		Spec: toolchainv1alpha1.IdlerSpec{TimeoutSeconds: 60},
+	}
+	namespaces := []string{"dev", "stage"}
+	usernames := []string{"alex"}
+	nsTmplSet := newNSTmplSet(test.MemberOperatorNs, "alex", "advanced", "abcde11", namespaces, usernames)
+	mur := newMUR("alex")
+	reconciler, _, _, _ := prepareReconcile(t, idler.Name, getHostCluster, idler, nsTmplSet, mur)
+	payloads := preparePayloads(t, reconciler, idler.Name, "", time.Now())
```

Comment:
could we use a different name of the variable instead of payloads since it shadows the struct in here https://github.com/codeready-toolchain/member-operator/blob/master/controllers/idler/idler_controller_test.go#L796